.build/travis_script.sh

```diff
@@ -76,4 +76,15 @@ printf "travis_fold:start:upgrade\nRun upgrade"
 pum upgrade -p qwat_prod -t qwat_sys.info -d $DELTA_DIRS
 echo "travis_fold:end:upgrade"
 
+# Extend qwat_prod with a customization
+printf "travis_fold:start:init-sire\nExtend database with ch_vd_sire"
+$TRAVIS_BUILD_DIR/customizations/sigip/init.sh -p qwat_prod -s 21781
```

Comment:
I'd name the file `init_sigip.sh` for consistency with `init_qwat.sh` and `init_sire.sh`.

Reply:
Right.